packages/rn-tester-e2e/package.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "rn-tester-e2e",
+    "version": "0.1.0",
+    "private": true,
```

```suggestion
	  "name": "@react-native/tester-e2e",
    "private": true,
	  "version": "0.0.1",
	  "license": "MIT",
	  "description": "React Native E2E tester app.",
	  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/rn-tester-e2e",
	  "repository": {
	    "type": "git",
	    "url": "git@github.com:facebook/react-native.git",
	    "directory": "packages/rn-tester-e2e"
	  },
```

suggestion to match rn-tester [as closely as possible](https://github.com/facebook/react-native/blob/main/packages/rn-tester/package.json)

---

done

---

done